pkg/bloombuild/planner/task.go

```diff
@@ -6,11 +6,16 @@ import (
 
 	"go.uber.org/atomic"
 
+	"github.com/grafana/loki/v3/pkg/bloombuild/common"
+	"github.com/grafana/loki/v3/pkg/bloombuild/planner/strategies"
 	"github.com/grafana/loki/v3/pkg/bloombuild/protos"
 )
 
 type QueueTask struct {
-	*protos.Task
+	*strategies.Task
+
+	// We use forSeries in ToProtoTask to get the chunks for the series in the gaps.
+	forSeries common.ClosableForSeries
```

Comment:
What's the difference between `common.ClosableForSeries` and `sharding.ForSeries` (which is used in `strategies.Task`?

Reply:
`common.ClosableForSeries` is

```
type ClosableForSeries interface {
	sharding.ForSeries
	Close() error
}
```

Added `ForSeries` alias to the common pkg and used it instead of `common.ClosableForSeries` and `sharding.ForSeries`
```
type ForSeries = sharding.ForSeries
```